packages/react-components/src/layout/src/Flex.tsx

```diff
@@ -154,13 +121,13 @@ export function InnerFlex({
     children,
     forwardedRef,
     ...rest
-}) {
+}: FlexProps): ReactElement {
     const isGapSupported = useIsGapSupported(noGap);
 
     // Normalize values until Chrome support `start` & `end`, https://developer.mozilla.org/en-US/docs/Web/CSS/align-items.
-    alignContent = alignContent && alignContent.replace("start", "flex-start").replace("end", "flex-end");
-    alignItems = alignItems && alignItems.replace("start", "flex-start").replace("end", "flex-end");
-    justifyContent = justifyContent && justifyContent.replace("start", "flex-start").replace("end", "flex-end");
+    const normalizedAlignContent = alignContent && alignContent.replace("start", "flex-start").replace("end", "flex-end");
+    const normalizedAlignItems = alignItems && alignItems.replace("start", "flex-start").replace("end", "flex-end");
```

Comment:
We do this in order to not mutate the component's props

Reply:
??